pwnlib/shellcraft/internal.py

```diff
@@ -88,10 +95,12 @@ def make_function(funcname, filename, directory):
     # but what would not have the right signature.
     # While we are at it, we insert the docstring too
     exec '''
-def wrap(template):
+def wrap(template, render_global):
     def %s(%s):
         %r
+        is_inside, render_global.is_inside = render_global.is_inside, True
```

Comment:
It is not actually easier to write. But sure, I could rewrite it if you prefer. :)